tests/integration-tests/tests/performance_tests/test_starccm.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import logging
+import boto3
+import json
+
+from remote_command_executor import RemoteCommandExecutor, RemoteCommandExecutionError
+
+# timeout in seconds
+INSTALLER_TIMEOUT = 1800
+SCHEDULER_TIMEOUT = 600
+STARCCM_SECRET_NAME = 'perf_integ_secret'
```

Comment:
The name of the secret should be more specific, .e.g "starccm_license_secret". 
the current name is not a good candidate because:
1. I could use the secrets not only for an integration tests but also for a demo
2. it's currently not referencing starccm at all, so when I see that secret in the console I don't know what's about.

Reply:
Furthermore, this constant is never used